### PR TITLE
Preserve original formatting in scope demo

### PR DESCRIPTION
**Pull request by bakkot**

### CODE REVIEW

**Comment by bakkot** [comment]
Fixes #15.

File: js/scope/main.js [hunk 1]
```diff
@@ -156,103 +156,68 @@ function mouseOverHandler(e) {
...
+function escapeHTML(str) {
+  return str.replace(/&/g, '&amp;').replace(/</g, '&lt;').replace(/>/g, '&gt;');
 }
 
-function WebGen() {
-  var webGen = new codegen.FormattedCodeGen;
+function collectIdentifiers(globalScope, locations) {
+  // returns the identifiers in source order.
+  var identifiers = [];
+  (function collect(scope) {
```
Comment (michaelficarra) [inline js/scope/main.js]: This can be done more elegantly without mutation.

**Review by michaelficarra — APPROVED** [review]
LGTM otherwise.

File: js/scope/main.js (hunk 1 above)
Comment (bakkot) [reply js/scope/main.js]: I dunno about *more* elegantly, but sure, done.

**Pull request merged**